### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by zenrabbit** [jennabraun/rdm.gradient#3]
1. why did you relevel?
2. can we replace site acronyms was a meaningful measure of site - ie aridity index? or label in some order from north-to-south or something? Right now, hard to tell what site differences 'mean' in the plots. Maybe just mean or total site level RDM as site level factor instead of labels - ie present all plots of site differences as a gradient - continuous or ordinal as you see fit.
3. Why abundance < 350?
4. I would treat shrub width as a covariate in models and put that plot in appendix for paper - maybe somehow showing it with RDM effects at the same time?
5. The regional slice plot of RDM greater under shrubs is super - not a gradient but closer than showing all the site-level plots.
6. species~RMD with fill = microsite key plot - assume this is abundance? or number of species? Best to have panels of three plots for figure 2 in paper - ie abudance, richness, and evenness by shrub-open faceted by region or ideally by gradient however you define - Given RDM not key driver maybe organize sites by aridity index - that is what Alex and I did for his gradient paper.
https://onlinelibrary.wiley.com/doi/abs/10.1111/jvs.12909
7. Rii plots good but really hard to see patterns.. can you stretch out x-axis somehow?
8. same - site-level plots hard to parse out.
9. I like shrub volume as a single composite measure - check Groot paper - lets all use the same way to estimate shrub canopy size ok?
10. glmmTMB is a super function. I have been using another way to check dispersion. In the R package AER you will find the function dispersiontest
11. to confirm, (1 | region/site) treats region and site are random effects and nests site inside region - and region has three levels? If so, this seems  a bit complex - did you test just a site vector that captures the gradient and ignored region too - region is arbitrary (but meaningful) and thus I would consider a single value that describes site ideal and simpler too. perhaps not possible, just musing here.
12. I love the AIC Model contrasts.
13 for the region-mean plots in model section, did you collapse sites into region - yes and the mean is the least-square means from models or arithmetic?  Also, again, I assume there was site heterogeneity.

**Ideas**
keep regions - three of the, mention, but put in Appendix

find value for each site and do all stats on site, random effect, and do a more typical graident analysis - heck as you know this can be ordinal - high versus low stress, continunous *aridity, or any other option

chose one 'level' of richness for paper

keep Rii and primary data too in paper

SEM great idea. great idea. I think it will show the exact same loadings though - ie shrub-open (ie microsite) is key driver and not RDM in predicting invert community.

**Comment by cjlortie** [comment]
Note - made all these notes before reading paper, but wanted to see workflow and thoughts before editing, amazing. I have so much to learn from you about stats. thanks for being patient with me JB!

**Comment by cjlortie** [comment]
[jvs.12909.pdf](https://github.com/jennabraun/rdm.gradient/files/4718600/jvs.12909.pdf)

**Comment by cjlortie** [comment]
checked - just aridity used as the x-axis. I do LIKE the three region model but think better in appendix.

**Comment by jennabraun** [comment]
1. relevel sets the reference level of the factor to the open site - so that in all models we get estimates for shrub effects. I think it makes it easier to understand results
2. rdm plots are separate but could try north-south,  none of these plots are in paper, just eda
3. there is one pitfall with 1200 of one species of ant and everything else is below 350, I think the mean is < 100. I'm calling it an outlier because I basically caught a colony. This took hours and hours to count because I had to check each ant and there were some other genera mixed in lol
4. There are some models at shrub microsites on looking for shrub size effects. There were not any so just a couple lines
7. These are already jittered I think - we don't have a nice even aridity gradient, it is three sets of three clusters. The gm smooth plot is the same data. 
9. I can test shrub volume, I'll will check to see what other insect papers to do too, maybe I will include both. For ground insects I don't know if width vs volume could be more relevant
10. cool
11. yes, if you look at the map and the environmental variables in Table one the sites fall into three clusters rather than a clean gradient. I tested site as a random effect VS site nested within region and the nested model is quantitatively a better model. There are no convergence issues so I don't understand how this could change complexity or interpretation in any way?

13. for RII vs RDM it is arithmetic mean and no regions, just the sites. 

For sure SEM would be great but lavaan doesn't support count data + I don't know how to do this so would delay paper a lot

**Comment by jennabraun** [comment]
@cjlortie on releveling: Reviewer for my ch2 paper says use open as reference "since they are your controls and combined in the model intercept term"

**Comment by cjlortie** [comment]
aha, never done.
thanks.
